src/app/providers/ThemeProvider/lib/ThemeContext.ts

```diff
@@ -0,0 +1,14 @@
+import {createContext} from "react";
+
+export const enum Themes {
```

Comment:
`enum` принято называть в единственном числе 

```suggestion
export const enum Theme {
```